ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py

```diff
@@ -289,12 +289,12 @@ def _check_celery():
 
         # try if celery works smoothly
         try:
-            guid = OpenvStorageHealthCheck.MACHINE_DETAILS.guid
-            machine_id = OpenvStorageHealthCheck.MACHINE_DETAILS.machine_id
+            guid = OpenvStorageHealthCheck.LOCAL_SR.guid
+            machine_id = OpenvStorageHealthCheck.LOCAL_SR.machine_id
```

Comment:
The healthcheck should not need any details of the environment. Therefore we do not use the API.